src/commands/build.ts

```diff
@@ -15,9 +15,13 @@
 // Only import type definitions here, otherwise this line will be included in
 // the JS output, triggering  the entire build library & its dependencies to
 // be loaded and parsed.
+import * as delTypeOnly from 'del';
+import * as path from 'path';
```

Comment:
Was there a reason to move this above? it probably doesn't make a huge difference performance-wise, but it contradicts the comment above and goes against our overall strategy of deferring loading all dependencies used during `run()`.